nucleus/admin/monitor/pom.xml

```diff
@@ -104,17 +105,17 @@
                 <artifactId>maven-site-plugin</artifactId>
                 <configuration>
                     <reportPlugins>
-                        <plugin>
+                        <!-- <plugin>
```

Comment:
Rather than comment stuff out can you just delete it please. Git history keeps a record of all changes.